kong-3.8.0-0.rockspec

```diff
@@ -205,6 +205,7 @@ build = {
     ["kong.tools.emmy_debugger"] = "kong/tools/emmy_debugger.lua",
     ["kong.tools.redis.schema"] = "kong/tools/redis/schema.lua",
     ["kong.tools.aws_stream"] = "kong/tools/aws_stream.lua",
+    ["kong.tools.cache"] = "kong/tools/cache.lua",
```

Comment:
let's just move to the exisitng `kong.tools.http`

Reply:
done